GUI_L02_2/Square.java

```diff
@@ -0,0 +1,10 @@
+package GUI_L02_2;
+
+class Square extends Rectangle {
+    private double side;
```

Comment:
This instance variable is not needed. You use the Rectangle's instance variables